### DISCUSSION TITLE
React-player doest not show video chapters.

### DISCUSSION THREAD

**Discussion by shahab7758** [cookpete/react-player#1799] [Ideas]
<ReactPlayer 
        width={'100%'}
        height={'auto'}
        url={`${videosCollection[selectedVideoId]?.url+videoResolution}.mp4`}   
        controls
        ref={playeref}
        playing  
        config={{
          file:{
            tracks: [
               { **kind: 'subtitles'**, src: '/subs/introduction.vtt', srcLang: 'en', default: true, label:'Locations'}
    // The same code snipet is working for **kind: 'subtitles'**
            ]
          }
        }}
 />

**Comment by thisKeeWord** [comment]
@cookpete any update on this?

**Comment by Sparh4wk** [comment]
would be awesome to support chapters in audio files as mp3, m4b etc as well. any chance for implementing this?
